### PR TITLE
Settings refactor, step one #755

### PR DESCRIPTION
**Pull request by umgelurgel**

### CODE REVIEW

File: pycon/settings.py
Comment (artcz) [inline pycon/settings.py]: I belive that by default is set to True, so this would be change in behaviour

File: pycon/settings.py
Comment (artcz) [inline pycon/settings.py]: this can be dropped for now, and later, if needed, we can probably go with dj-database-url anyway.

File: pycon/settings.py
Comment (artcz) [inline pycon/settings.py:317]: do we need this as env setting?

File: requirements/dev.txt
Comment (artcz) [inline requirements/dev.txt]: this is probably no longer needed because of python3-openid below

File: requirements/dev.txt
Comment (artcz) [inline requirements/dev.txt]: why did it drop [security]?

**Comment by artcz** [comment]
LGTM, but we need to check it staging before merge, so i'm waiting for that with final approval.

File: pycon/settings.py
Comment (umgelurgel) [reply pycon/settings.py:317]: Until there's separate modules for settings, where additional settings can be defined, I can't think of another way of doing it. Once there's a separate `prod`, `staging`, `dev` and `test` settings, this can be dropped.

File: requirements/dev.txt
Comment (umgelurgel) [reply requirements/dev.txt]: No clue - it's the `pip-compile` magic :)

**Review by alexsavio — APPROVED** [review]
LGTM

**Review by artcz — APPROVED** [review]
LGTM

**Pull request merged**
